pkg/admission/cluster_webhook.go

```diff
@@ -47,6 +47,8 @@ func DefaultCluster(ctx context.Context, _ client.Client, obj runtime.Object) er
 	if !ok {
 		return nil
 	}
+	cluster.SetDefaultTokenValidityIfNeeded()
```

Comment:
The `kubebuilder:default` annotation defines a default value for the `MaxTokenValidity`. Do we need to default it again? Could you validate with a test if the defaulting works with just the annotation set?

Reply:
We don't need to defaulting webhook if we use the `kubebuilder` default annotation

You can see this in the CRD manifest that is generated -

https://github.com/cloudoperators/greenhouse/pull/634/files#diff-51013bf768fe937b255fe3c3bb649b25a64efd134fc1293c59dfd3eb7d6a0046R1097-R1100

Reply:
I have doubts so I will check it.

Reply:
ok 😸

Reply:
I've checked it. The `kubebuilder:default` is not enough so we should have it here.

Reply:
Done